docs/rules/jsx-handler-names.md

```diff
@@ -43,6 +44,7 @@ Examples of **correct** code for this rule:
 - `eventHandlerPropPrefix`: Prefix for props that are used as event handlers. Defaults to `on`
 - `checkLocalVariables`: Determines whether event handlers stored as local variables are checked. Defaults to `false`
 - `checkInlineFunction`: Determines whether event handlers set as inline functions are checked. Defaults to `false`
+- `ignoreComponentNames`: Array of regex, when matched with component name, ignores the rule on that component. Defaults to `[]`
```

Comment:
it's a bad idea for eslint plugins to support regex strings from config; we can support glob strings, though.

Reply:
Curious to know the rationale behind it.
Unrelated to this, I recently contributed to [eslint/no-restricted-exports](https://github.com/eslint/eslint/pull/18431), where regex was preferred over glob string.

Reply:
regexes are a CVE magnet, every time they exist. eslint shouldn't have accepted a pattern string either, imo.

Reply:
Updated to use glob strings